test/workspaces/pip-app-with-python-markers/requirements.txt

```diff
@@ -1,6 +1,7 @@
 amqp==2.4.2
 apscheduler==3.6.0
 asn1crypto==0.24.0
+lazy-object-proxy==1.6.0
```

Comment:
`astroid` depends on `lazy-object-proxy`. When not specifying, it defaults to 1.7.0 which is not supported by python2.7